### PR TITLE
add user info to user view

### PR DESCRIPTION
**Pull request by xuwhite**
additionally fixing the logs of user updates

### CODE REVIEW

File: includes/pages/admin_user.php [hunk 1]
```diff
@@ -26,7 +26,8 @@ function admin_user()
     $html = '';
     $goodie = GoodieType::from(config('goodie_type'));
     $goodie_enabled = $goodie !== GoodieType::None;
-    $goodie_tshirt = $goodie === GoodieType::Tshirt;
+    $goodie_tshirt = $goodie == GoodieType::Tshirt;
```
Comment (MyIgel) [inline includes/pages/admin_user.php:29]: ???

File: includes/pages/admin_user.php [hunk 2]
```diff
@@ -98,16 +113,17 @@ function admin_user()
...
         $html .= html_options('eAktiv', $options, $user_source->state->active) . '</td></tr>' . "\n";
 
-        // Aktiv erzwingen
+        // Forced active?
         if (auth()->can('admin_active')) {
             $html .= '  <tr><td>' . __('Force active') . '</td><td>' . "\n";
             $html .= html_options('force_active', $options, $user_source->state->force_active) . '</td></tr>' . "\n";
         }
 
+
```
Comment (MyIgel) [inline includes/pages/admin_user.php:126]: Interesting new line?

File: includes/view/User_view.php [hunk 3]
```diff
@@ -543,7 +542,10 @@ function User_view(
             : ''
         )
         . htmlspecialchars($user_source->name)
-        . (config('enable_user_name') ? ' <small>' . $user_name . '</small>' : ''),
+        . (config('enable_user_name') ? ' <small>' . $user_name . '</small>' : '')
+        . ((auth()->can('user.info.show') && $user_source->state->user_info)
+        ? (' <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip" title="'
+        . $user_source->state->user_info . '"></span>') : ''),
```
Comment (MyIgel) [inline includes/view/User_view.php:548]: Should be escaped too

Comment (MyIgel) [reply includes/view/User_view.php:548]: Also adding `<small>[content]</small>` helps it a bit to blend in :D

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: includes/pages/admin_user.php (hunk 1 above)
Comment (xuwhite) [reply includes/pages/admin_user.php:29]: ill revert that

**Pull request merged**